src/styles/button.css

```diff
@@ -0,0 +1,27 @@
+
+.product__button {
+  display: block;
+  padding-top: 16px;
+}
+
+.button {
```

Comment:
The same way here – if you have a separate BEM-block, move it to a separate file